reactpy_mkdocs/reactpy_frame.py

```diff
@@ -0,0 +1,123 @@
+from pathlib import Path
+from io import StringIO
+from traceback import format_exc
+from logging import getLogger
+from typing import Any, Callable
+from urllib.parse import parse_qs
+
+import reactpy
+from reactpy import component, use_location, html, use_state
+from reactpy.types import ComponentType
+
+log = getLogger(__name__)
+
+REACTPY_RUN = reactpy.run
```

Comment:
Needs a comment to describe why this is being declared as a global, rather than `from reactpy import run as REACTPY_RUN`

Reply:
I think this can be renamed to `ORIGINAL_REACTPY_RUN` too. We patch this on the fly below so we [can capture components](https://github.com/reactive-python/reactpy-mkdocs/pull/1/files#diff-6e9f957b6c407328a17b24e88fddebdd021fb0f49eb3958627cc26dc257281b1R52-R56) from example code.